pkg/operator/ceph/object/topic/provisioner.go

```diff
@@ -167,6 +169,13 @@ func createTopic(p provisioner, topic *cephv1.CephBucketTopic) (*string, error)
 		attr["push-endpoint"] = &topic.Spec.Endpoint.HTTP.URI
 		verifySSL = strconv.FormatBool(!topic.Spec.Endpoint.HTTP.DisableVerifySSL)
 		attr["verify-ssl"] = &verifySSL
+		if p.clusterInfo.CephVersion.IsAtLeast(version.Quincy) {
+			cloudEvents := strconv.FormatBool(topic.Spec.Endpoint.HTTP.CloudEvents)
+			attr["cloudevents"] = &cloudEvents
+		} else if topic.Spec.Endpoint.HTTP.CloudEvents {
+			logger.Warning("HTTP CloudEvents endpoint needs cluster to be Quincy or newer")
+
+		}
```

Comment:
Unit test coverage? I see 4 cases: Quincy yes/no, Enabled yes/no.

Reply:
i don't have provisioner unit tests

Reply:
Let's make a note to get those created at some point then.

Reply:
How about splitting out into a helper method, which could be easily unit tested?